src/Http/Routing/test/UnitTests/Matching/CandidateSetTest.cs

```diff
@@ -325,10 +325,11 @@ public void ExpandEndpoint_ThrowsForDuplicateScore()
             var ex = Assert.Throws<InvalidOperationException>(() => candidateSet.ExpandEndpoint(0, Array.Empty<Endpoint>(), comparer));
 
             // Assert
-            Assert.Equal(@"
-Using ExpandEndpoint requires that the replaced endpoint have a unique priority. The following endpoints were found with the same priority:
-test: /0
-test: /1"
+            Assert.Equal(@"Using ExpandEndpoint requires that the replaced endpoint have a unique priority. The following endpoints were found with the same priority:" +
+                Environment.NewLine +
+                "test: /0" +
+                Environment.NewLine +
+                "test: /1"
```

Comment:
FYI there's also an extra parameter you can pass that specifies whether to ignore newlin differences. I'm not sure how this got missed by the CI when I wrote the test, but #YOLO

Reply:
It didn't get missed, is that helix isn't in your face about failures. I'll be changing that shortly with https://github.com/aspnet/AspNetCore/pull/9187.